clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -975,7 +975,7 @@ public void onFailure(RuntimeException e) {
                     }
                 }
             } catch (InterruptedException | InterruptException e) {
-                Thread.interrupted();
+                Thread.currentThread().interrupt();
```

Comment:
The intent was to clear the interrupt, which is what `interrupted` accomplishes. Are you suggesting that we should be resetting the interrupt?